echidna/core/spectra.py

```diff
@@ -77,6 +80,7 @@ def fill(self, energy, radius, time, weight=1.0):
         radial_bin = (radius - self._radial_low) / (self._radial_high - self._radial_low) * self._radial_bins
         time_bin = (time - self._time_low) / (self._time_high - self._time_low) * self._time_bins
         self._data[energy_bin, radial_bin, time_bin] += weight
+        self._raw_events += 1
```

Comment:
@jwaterfield, I think this is fine. Can you just add a note somewhere, preferably that will show up in the documentation, that this is based on the assumption that only one raw event is added for each call of `Spectra._fill`. I know it seems pretty obvious, but `Spectra._fill` can be used to add multiple events by changing the weight, so I just think it would be useful to clarify this assumption somewhere.